app/javascript/packs/providers/HelloWorldProvider.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Provider } from 'react-redux'
+import { getStore } from 'providers/Store'
```

Comment:
I think you can use it this way: `import store, { getStore } from 'providers/Store'